tests/virt_autotest/alp_guest_installation.pm

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: This file handles the guest installation for ALP.
+#          It supports configuring vm profiles with template.
+#
+# Maintainer: Alice <xlai@suse.com>, or VT squad <qe-virt@suse.de>
+
+package alp_guest_installation;
+
+use base unified_guest_installation;
+use strict;
+use warnings;
+use testapi;
+use Data::Dumper;
+
+sub edit_guest_profile_with_template {
+    my ($self, $guest_name) = @_;
+
+    # Initialize config
+    my %_guest_profile = %{$concurrent_guest_installations::guest_instances_profiles{$guest_name}};
+
+    # Handle guest_boot_settings
+    my $_boot_value = '';
+    $_boot_value = 'uefi' if (get_var('VIRT_UEFI_GUEST_INSTALL', ''));
+    $_boot_value = 'firmware=efi' if (get_var('VIRT_SEV_ES_GUEST_INSTALL', ''));
+    $_guest_profile{guest_boot_settings} =~ s/##guest_boot_settings##/$_boot_value/g;
+
+    # Handle guest_storage_others
+    my $_backing_disk_name = get_var('HDD_1', '');
+    $_backing_disk_name =~ /([^\/]+)\.([^\/\.]+)$/m;
+    my $_new_back_name = "$1-back.$2";
+    my $_back_format = $2;
+    $_guest_profile{guest_storage_others} =~ s/##backing_disk_name##/$_new_back_name/g;
+    $_guest_profile{guest_storage_others} =~ s/##backing_format##/$_back_format/g;
+
+    # Handle version
+    my $_version = get_var('VERSION', '');
+    $_version =~ /[a-zA-Z]+-([0-9\.]+)/m;
+    $_version = $1;
+    $_guest_profile{guest_version} =~ s/##guest_version##/$_version/g;
+
+    # Handle build
+    my $_build = get_var('BUILD', '');
```

Comment:
The same concern here with ```BUILD```

Reply:
Please see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16839/files#r1168188968.